jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-jbpm-schema.sql

```diff
@@ -768,8 +768,7 @@
     create index IDX_TaskEvent_taskId on TaskEvent (taskId);
     create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
 
-    create index IDX_EventTypes_element ON EventTypes(element);
-    create index IDX_EventTypes_compound ON EventTypes(InstanceId, element);
+    create index IDX_EventTypes_IdElement ON EventTypes(InstanceId, element);
```

Comment:
Maybe Im missing something but why we need to change the name of the index from compount to IdElement?. I believe the only required changes is to align the JPA entity with the script.

Reply:
Hi @fjtirado,

yes you are correct. The only required changes is to align the JPA entity. But because I see that ALL DBMS sql scripts has this name (and only 'mysql' and 'mysqlinnodb' other name), I change also the name for 'mysql' and 'mysqlinnodb' sql scripts. Let me know if this change is Not accepted, and if so I will do another PR only with the changes on the JPA entity.

Thank you,
Daniele Mammarella

Reply:
Thats fine, just trying to find out why, what about the other index that is deleted? Notice that queries which uses only element can still benefit from it

Reply:
Hi @fjtirado ,

I see what is done for other DBMS.
Look for example:
1. https://github.com/kiegroup/jbpm/blob/dc1c97cb52a9c145058182456704e4030a158ba3/jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/rhpam-7.13-to-7.13.1.sql#L3
2. https://github.com/kiegroup/jbpm/blob/dc1c97cb52a9c145058182456704e4030a158ba3/jbpm-db-scripts/src/main/resources/db/ddl-scripts/postgresql/postgresql-bytea-jbpm-schema.sql#L117
3. https://github.com/kiegroup/jbpm/commit/830bbb9717f490cc0f216ca9eabd00fc1b559791

Thank you,
Daniele Mammarella

Reply:
Thanks for the explanation

Reply:
Hi @fjtirado ,

Let me know if you would like to have on this PR also the diff that will align also the other index named: 'IDX_EventTypes_Id', as I said in my previous comment, that were wrongly (I suppose) be removed from that commit.
I have the diff ready on a patch_file. If you agree with me, I will commit and push also this diff.

Thank you,
Daniele Mammarella

Reply:
I agree to do all the aligment needed with the JIRA

Reply:
Hi @fjtirado ,

ok thank you. I just push the new commit:
https://github.com/kiegroup/jbpm/pull/2395/commits/30fba8a4c592a9ec6221089db1cb6a8746764fa0
on this Pull Request.

Waiting for your approval.

Thank you,
Daniele Mammarella